crates/db_connection_pool/src/snowflakepool.rs

```diff
@@ -124,6 +113,74 @@ impl SnowflakeConnectionPool {
     }
 }
 
+fn init_snowflake_api(
+    params: &Arc<Option<HashMap<String, String>>>,
+    secret: &Option<Secret>,
+) -> Result<SnowflakeApi> {
+    let username = get_param(params, secret, "username")
+        .context(MissingRequiredSecretSnafu { name: "username" })?;
+    let account = get_param(params, secret, "account")
+        .context(MissingRequiredSecretSnafu { name: "account" })?;
+    // account identifier can be in <orgname.account_name> format but API requires it as <orgname-account_name>
+    let account = account.replace('.', "-");
+
+    let warehouse = get_param(params, secret, "snowflake_warehouse");
+    let role = get_param(params, secret, "snowflake_role");
+    let auth_type = get_param(params, secret, "snowflake_auth_type")
+        .unwrap_or_else(|| "auth_snowflake".to_string())
```

Comment:
`auth_snowflake` for default username/password auth was selected to be consistent with Snowflake naming: it is called `snowflake` in `snowsql` tool, `go driver`, ADBC driver, etc

Reply:
Then it should just be `snowflake` then, not `auth_snowflake`?

Reply:
@phillipleblanc - thank you - I'll update to just `snowflake` (it is what `snowsql` is using). Snowflake ADBC driver naming is `auth_snowflake`, but just `snowflake` is better IMO (same for `keypair`)
![image](https://github.com/spiceai/spiceai/assets/981580/8656470a-8ac9-4128-910c-36848ffab70c)
![image](https://github.com/spiceai/spiceai/assets/981580/0329d7fe-3e9a-4092-9e1b-04b57ee43427)

Reply:
Updated - thank you